setup.py

```diff
@@ -4,29 +4,17 @@
 """Palo Alto Networks Cloud Python SDK setup script."""
 
 from setuptools import setup, find_packages
-import socket
 
 with open('README.rst') as readme_file:
     readme = readme_file.read()
 
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-if hasattr(socket, 'ssl'):
-    requirements = [
-        'requests',
-        'requests-oauthlib',
-        'cffi',
-        'cryptography'
-    ]
-else:
-    requirements = [
-        'requests',
-        'requests-oauthlib',
-        'pyopenssl',
-        'cffi',
-        'cryptography'
-    ]
+requirements = [
```

Comment:
I don't remember why requirements is duplicated.  but this is missing six (but maybe we can remove that dependency).  can we just populate this by reading requirements.txt?

Reply:
I'll look into it. I agree that reading from requirements.txt would be the cleanest approach.